### DISCUSSION TITLE
How to delete all revisions of an entire collection?

### DISCUSSION THREAD

**Discussion by Yuzal** [ravendb/ravendb#12526] [Q&A] [answered]
Is it possible to delete all revisions of an entire collection?
I know there is an endpoint to delete all revisions with document ids... but can i use this as well to delete entire collections?

**Comment by ayende** [comment, marked as answer]
You can setup the revisions limit as 0, and then click on enforce configuration in Settings > Documents Revisions.

![image](https://user-images.githubusercontent.com/116915/126326308-c56ea21c-c4cb-4fd9-8419-dbd0789de464.png)

**Reply by Yuzal** [reply]
Perfect!
This solution is absolut fine for our case.
Thank you
/closed
